cli/command/stack/remove.go

```diff
@@ -55,10 +56,20 @@ func runRemove(dockerCli command.Cli, opts removeOptions) error {
 			return err
 		}
 
-		configs, err := getStackConfigs(ctx, client, namespace)
+		var configs []swarm.Config
+
+		version, err := client.ServerVersion(ctx)
 		if err != nil {
 			return err
 		}
+		if versions.LessThan(version.APIVersion, "1.30") {
+			fmt.Fprintf(dockerCli.Err(), "WARNING: requires API version 1.30, but the Docker daemon API version is %s", version.APIVersion)
```

Comment:
What requires API version 1.30? If i saw this message from `docker stack rm` I would be confused. I think we need to include that this is about "configs" only.